charts/artifact-hub/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: artifact-hub
 description: Artifact Hub is a web-based application that enables finding, installing, and publishing Kubernetes packages.
 type: application
-version: 1.15.0
```

Comment:
We need to bump the version to make the Helm linter happy. Can you update it to `1.15.1-0`?

Reply:
Hello, done